sdk-setup/src/mb2

```diff
@@ -1775,6 +1795,7 @@ run_build() {
             "$TMP_SPEC" \
             "$@"
```

Comment:
Now that it is not the last command in the function, we need to handle possible failure:

```suggestion
            "$@" \
            || return
```